### PR TITLE
feat: setup accessibility testing and linting

### LINKED ISSUE #63: Define accessibility testing

### PR DESCRIPTION
**Pull request by larsrickert**
closes #63 

## Description

- Setup Playwright for a11y testing
- Setup eslint plugin to show a11y violations while development

## Checklist

- [x] The added / edited code has been documented with [JSDoc](https://jsdoc.app/about-getting-started)
- [x] All changes are documented in the documentation app (folder `apps/docs`)
- [x] If a new component is added, at least one [Playwright screenshot test](https://github.com/SchwarzIT/onyx/actions/workflows/playwright-screenshots.yml) is added
- [x] A changeset is added with `npx changeset add` if your changes should be released as npm package (because they affect the library usage)

### CODE REVIEW

File: packages/sit-onyx/src/components/TestInput/TestInput.spec.tsx [hunk 1]
```diff
@@ -1,16 +1,22 @@
-import { expect, test } from "@playwright/experimental-ct-vue";
+import { expect, test } from "../../playwright-axe";
```
Comment (JoCa96) [inline packages/sit-onyx/src/components/TestInput/TestInput.spec.tsx:1]: Can we enforce that the import always comes from the fixture?

Comment (larsrickert) [reply packages/sit-onyx/src/components/TestInput/TestInput.spec.tsx:1]: Probably only with a custom eslint rule, let me check that

File: packages/sit-onyx/src/components/TestInput/TestInput.vue [hunk 2]
```diff
@@ -174,9 +174,6 @@ watch(
     color: darkred;
     margin: 0;
   }
-  &__info {
-    color: grey;
```
Comment (BoppLi) [inline packages/sit-onyx/src/components/TestInput/TestInput.vue:178]: was the contrast not good enough anymore? :(

Comment (larsrickert) [reply packages/sit-onyx/src/components/TestInput/TestInput.vue:178]: Nope :D

File: packages/sit-onyx/src/components/TestInput/TestInput.spec.tsx (hunk 1 above)
Comment (larsrickert) [reply packages/sit-onyx/src/components/TestInput/TestInput.spec.tsx:1]: Done :)

**Review by jannick-ux — APPROVED** [review]

**Review by JoCa96 — DISMISSED** [review]

**Review by JoCa96 — APPROVED** [review]

**Review by jannick-ux — APPROVED** [review]

**Pull request merged**